### PR TITLE
fix: "dev:exec" command description

### PR DESCRIPTION
**Pull request by sean-roberts**
Descriptions don't support \n line breaks so the description is currently just "Exec Command" Removing that prefix and line break makes the description more consistent with our other command description patterns.

shoutout to @rjbeers for catching it

### CODE REVIEW

**Review by serhalp — APPROVED** [review]
Thanks for fixing!

**Review by serhalp — DISMISSED** [review]

**Review by TylerBarnes — DISMISSED** [review]

**Review by codebyuma — DISMISSED** [review]
Sorry for the late review! 

Looks good to me, though I left a comment to fix some style guide things that Vale is complaining about. I know it's from existing content but since you're touching these lines anyway, I figure we can fix them at the same time 😄 

🚢

**Comment by serhalp** [comment]
bump @sean-roberts 😁

**Comment by sean-roberts** [comment]
> bump @sean-roberts 😁

ooof I did not see this, thanks for the tag!

File: docs/commands/dev.md [hunk 1]
```diff
@@ -57,7 +57,6 @@ BROWSER=none netlify dev # disable browser auto opening
 ---
 ## `dev:exec`
 
-Exec command
 Runs a command within the netlify dev environment, e.g. with env variables from any installed addons
```
Comment (serhalp) [inline docs/commands/dev.md:60]: Missed one here (I'll push it):
```suggestion
Runs a command within the netlify dev environment. For example, with environment variables from any installed add-ons
```

**Review by serhalp — DISMISSED** [review]

**Review by codebyuma — DISMISSED** [review]

**Review by serhalp — APPROVED** [review]

**Pull request merged**